### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by josecamachop** [jogonmar/MVBatch#13]
Routines in modelling for JM: reconstructX, plotOverall, ploton (just check), plotoff, plot3Dvar, plot3D_batchtools, plot_onwarp, plot_discv2 (check list of outputs), plot_contributions, contrbT2, contrbSPE

**Comment by jogonmar** [comment]
I believe that what you mean is that you have moved the functions listed above to the modeling folder, isn´t it?

**Comment by josecamachop** [comment]
No, this is a list of functions, already in the modeling folder, that were created by you. I have added the documentation agreed to the rest of routines in that folder, and you need to do the same for these.

**Comment by josecamachop** [comment]
Hi Jose, you seem to have finished this issue. If so, please close it.
